src/cmd/tools.go

```diff
@@ -80,6 +80,10 @@ var archiverDecompressCmd = &cobra.Command{
 				if strings.HasSuffix(file.Name(), "tar.zst") {
 					if err := archiver.Unarchive(filepath.Join(layersDir, file.Name()), layersDir); err != nil {
```

Comment:
While I don't disagree, what would the be expected behavior be? Should any functionality be mirrored with some configuration with `zarf create package`?

Reply:
On second thought, I think this is fine enough for now because the follow commands will re-create the original package.
```
zarf tools archiver decompress {Package} {PATH}
zarf tools archiver compress {PATH}
```